### PR TITLE
Combined client chart v2

### PR DESCRIPTION
**Pull request by esbie**
screen shot 2016-04-01 at 12 29 28 pm
Adds a chart below each router summary.
Modifies UpdateableChart to support polling via the metricsCollector.

### CODE REVIEW

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+var CombinedClientGraph = (function() {
+  function clientToMetric(client) {
+    return {name: client, color: ""}; //TODO: move to clientName only after v2 migration
+  }
+
+  return function(metricsCollector, routerName, $root, colors) {
+    var query = Query.clientQuery().withRouter(routerName).withMetric("requests").build();
+
+    var timeseriesParams = function(metricName) {
```
Comment (rmars) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js]: argument metricName is unused

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        minValue: 0,
+        grid: {
+          strokeStyle: 'rgba(255,255,255,0.3)',
+          fillStyle: 'rgba(0,0,0,0)',
+          verticalSections: 1,
+          millisPerLine: 10000,
+          borderVisible: false
+        },
+        labels: {
+          fillStyle: '#878787',
```
Comment (klingerf) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js]: Totally minor, but why use rgba for the grid.fillStyle, and hex for the labels.fillStyle?

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_controller.js [hunk 3]
```diff
@@ -19,6 +80,7 @@ var RouterController = (function () {
     var routerContainerEls = initializeRouterContainers(selectedRouter, routers, $parentContainer, templates.container);
 
     var routerSummaries = [];
+    var routerGraphs = [];
```
Comment (rmars) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_controller.js]: now that each listener registers itself with the metrics collector, I don't think we need to keep track of all these any more

**Comment by klingerf** [comment]
:star: :turtle: Looks good to me, modulo @rmars's comment.

**Comment by rmars** [comment]
:+1: :octopus: :dog: yay graphs!

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js (hunk 2 above)
Comment (esbie) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js]: good point, I'll migrate to hex

**Pull request merged**